srcpkgs/boost/template

```diff
@@ -104,11 +103,327 @@ boost-build_package() {
 }
 boost-devel_package() {
 	noverifyrdeps=yes
-	depends="boost-${version}_${revision}"
+	depends="libboost_unit_test_framework1.69-${version}_${revision}
+	 libboost_date_time1.69-${version}_${revision}
+	 libboost_atomic1.69-${version}_${revision}
+	 libboost_contract1.69-${version}_${revision}
+	 libboost_random1.69-${version}_${revision}
+	 libboost_locale1.69-${version}_${revision}
+	 libboost_stacktrace_basic1.69-${version}_${revision}
+	 libboost_filesystem1.69-${version}_${revision}
+	 libboost_stacktrace_addr2line1.69-${version}_${revision}
+	 boost-${version}_${revision} libboost_wave1.69-${version}_${revision}
+	 libboost_type_erasure1.69-${version}_${revision}
+	 boost-python1.69-${version}_${revision}
+	 libboost_math_tr1f1.69-${version}_${revision}
+	 libboost_math_tr11.69-${version}_${revision} boost-devel-${version}_${revision}
+	 libboost_iostreams1.69-${version}_${revision}
+	 libboost_regex1.69-${version}_${revision}
+	 libboost_system1.69-${version}_${revision}
+	 libboost_chrono1.69-${version}_${revision}
+	 libboost_math_c991.69-${version}_${revision}
+	 libboost_graph1.69-${version}_${revision}
+	 libboost_log_setup1.69-${version}_${revision}
+	 libboost_fiber1.69-${version}_${revision}
+	 libboost_container1.69-${version}_${revision}
+	 libboost_context1.69-${version}_${revision}
+	 libboost_log1.69-${version}_${revision}
+	 libboost_timer1.69-${version}_${revision}
+	 boost-python3-1.69-${version}_${revision}
+	 libboost_thread1.69-${version}_${revision}
+	 libboost_prg_exec_monitor1.69-${version}_${revision}
+	 libboost_math_c99l1.69-${version}_${revision}
+	 libboost_program_options1.69-${version}_${revision}
+	 libboost_wserialization1.69-${version}_${revision}
+	 libboost_serialization1.69-${version}_${revision}
+	 libboost_stacktrace_noop1.69-${version}_${revision}
+	 libboost_coroutine1.69-${version}_${revision}
+	 libboost_math_c99f1.69-${version}_${revision}
+	 libboost_math_tr1l1.69-${version}_${revision}"
 	short_desc+=" - development files"
 	pkg_install() {
 		vmove usr/include
 		vmove "usr/lib/*.a"
 		vmove "usr/lib/*.so"
 	}
 }
+
+libboost_atomic1.69_package() {
+	depends="boost-${version}_${revision}"
```

Comment:
Is it necessary to depend on `boost` ? Most cases where a libXver package is created the main package becomes a holder for cmdline utilities and tooling

Reply:
boost currently only contains the LICENSE file, nothing else.